### DISCUSSION TITLE
Media MTX server migration from rtsp_simple_server

### DISCUSSION THREAD

**Discussion by Rambabu-Tirumalasetty** [bluenviron/mediamtx#3729] [Questions]
### Question

Hi Team,

Greetings for the day!

We have implemented a streaming solution based on rtsp_simple_server and pusher code. We were able to stream USB camera through this.

Now we have a plan of migrating to mediaMTX from rtsp_simple_server. So, we ran pre-built binary and tested our application.
We could see connection establishment and recording the stream. But through ffplay command or vlc I'm not able to view the stream.

Can you please provide inputs on this?

Thanks,
Rambabu.
[mediamtx_txt.txt](https://github.com/user-attachments/files/16862785/mediamtx_txt.txt)

**Comment by aler9** [comment]
Hello, logs show that you are correctly publishing a H264 stream to the server.

If readers are unable to read the stream, it might depend on the content of the stream.

In order to replicate the issue, you need to provide a network dump, that can be generated in this way:

1. Download wireshark (https://www.wireshark.org/)
2. Start capturing on the interface used for exchanging packets * If the server and the external hardware or software are both installed on your pc, the interface is probably "loopback", otherwise it's the one of your network card.
3. Start the server and replicate the issue
4. Stop capturing, save the result in .pcap format
5. Attach
